docs/source/user/faq.rst

```diff
@@ -10,16 +10,18 @@ Programming
 Can I pass a function as an argument to a jitted function?
 ----------------------------------------------------------
 
-You can't, but in many cases you can use a closure to emulate it.
-For example, this example::
+As of Numba 0.39, you can, so long as the function argument has also been
+JIT-compiled::
```

Comment:
Perhaps update the code sample(s) so that the function passed in is called e.g. `jitted_g_function` ?